test/check-networking

```diff
@@ -173,4 +174,14 @@ class TestNetworking(MachineCase):
         b.enter_page("networking")
         b.wait_not_in_text("#networking-interfaces", "tbridge")
 
-test_main()
+if 'TEST_OS' in os.environ and os.environ['TEST_OS'] == "rhel-7":
+
+    # HACK - We lose all connectivity to the VM for about 30 seconds
+    #        after changing the config of the additional network
+    #        interfaces.
+    #
+    # XXX - file bug
```

Comment:
Sounds pretty wild. I guess lets file this bug before we merge this pull request.